store/src/main/java/com/dropbox/android/external/store4/impl/SourceOfTruth.kt

```diff
@@ -25,7 +25,7 @@ import kotlinx.coroutines.flow.flow
  * [RealStore] implementation to avoid dispatching values to downstream while
  * a write is in progress.
  */
-internal interface SourceOfTruth<Key, Input, Output> {
+ interface SourceOfTruth<Key, Input, Output> {
```

Comment:
I'm not sure you really need to expose this type. Can you try to make it internal again